utils/formatCountyData.ts

```diff
@@ -1,61 +1,157 @@
 import formatGraph from './formatGraph'
 
 type CaseType = {
-  cases: number
-  deaths: number
-  recovered: number
-  active: number
-  date: Date
+  ['cumul_cases']: number
+  date: string
+}
+
+type DeathType = {
+  ['cumul_deaths']: number
+  date: string
 }
 
 type CountyDataType = {
   name: string
-  population: number
-  cases: Array<CaseType>
+  ['update_time']: string
+  series: {
+    cases: Array<CaseType>
+    deaths: Array<DeathType>
+  }
 }
 
 type GraphDataType = {
   label: string
   confirmedTransition: number
   cumulative: number
+  deathTransition: number
+  deathCumulative: number
 }
 
-type CountyDataFormattedType = {
+type CountiesFormattedType = {
   name: string
   population: number
+  color: string
   graph: Array<GraphDataType>
-  lastUpdatedAt: Date
+  lastUpdatedAt: string
+}
+
+interface TotalsFormattedType extends CountiesFormattedType {
+  totalPopulation: number
+  cases: Array<GraphDataType>
+}
+
+interface CountyInterface<Type> {
+  solano: Type
+  alameda: Type
+  ['santa_clara']: Type
+  ['san_francisco']: Type
+  ['contra_costa']: Type
+  ['san_mateo']: Type
+  sonoma: Type
+  napa: Type
+  marin: Type
+}
+
+type CountyCollection = Partial<CountyInterface<CountiesFormattedType>> & {
+  totals: TotalsFormattedType
+}
+
+const COUNTY_POPULATIONS: CountyInterface<number> = {
```

Comment:
We had population data in formatCountyDataVTwo.ts. We want to avoid double maintenance and use the same population data for all calculation in different charts.